make-util.js

```diff
@@ -318,4 +320,115 @@ var getExternals = function (externals, destRoot) {
         });
     }
 }
-exports.getExternals = getExternals;
\ No newline at end of file
+exports.getExternals = getExternals;
+
+//------------------------------------------------------------------------------
+// task.json functions
+//------------------------------------------------------------------------------
+
+var createResjson = function (task, taskPath) {
+    var resources = {};
+    if (task.hasOwnProperty('friendlyName')) {
+        resources['loc.friendlyName'] = task.friendlyName;
```

Comment:
these were const strings before, but for readability i just hardcoded them. if you have a preference the other way, i don't mind moving them back to const strings.